src/Router.jsx

```diff
@@ -12,10 +12,10 @@ import LectureVideo from './pages/LectureVideo/LectureVideo';
 import CreatorCenter from './pages/CreatorCenter/CreatorCenter';
 import Main from './pages/Main/Main';
 
-const Router = () => {
+const Router = props => {
```

Comment:
```suggestion
const Router =() => {
```
컴포넌트에서 부모로부터 받는 props가 없다면 인자에서 제거해주세요!